### PR TITLE
Added free resources for Python & R

### PR DESCRIPTION
**Pull request by ryzalk**
Hi there. Just added some links for books that beginners can jump into for R/Python :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,6 +33,7 @@ With Applications in R](http://fs2.american.edu/alberto/www/analytics/ISLRLectur
 - [Deep reinforcement learning Course](https://github.com/simoninithomas/Deep_reinforcement_learning_Course)
 - [Simple Reinforcement Learning with Tensorflow](https://medium.com/emergent-future/simple-reinforcement-learning-with-tensorflow-part-0-q-learning-with-tables-and-neural-networks-d195264329d0)
 - [Reinforcement Learning by Georgia Tech](https://www.udacity.com/course/reinforcement-learning--ud600)
+- [Advanced R](http://adv-r.had.co.nz/)
```
Comment (alvations) [inline README.md:39]: Hmmm, I think we should have a `### Language` section for things like this.

File: README.md [hunk 2]
```diff
@@ -17,6 +17,8 @@ Knowledge base for DSSG.
 - [MIT: The Analytics Edge](https://www.edx.org/course/the-analytics-edge)
 - [Andrew Ng: Machine Learning](https://www.coursera.org/learn/machine-learning)
 - [CMU: Machine Learning](http://www.cs.cmu.edu/~tom/10701_sp11/lectures.shtml)
+- [Python Data Science Handbook](https://jakevdp.github.io/PythonDataScienceHandbook/)
```
Comment (alvations) [inline README.md]: 👍

File: README.md [hunk 3]
```diff
@@ -17,6 +17,8 @@ Knowledge base for DSSG.
...
+- [R for Data Science](https://r4ds.had.co.nz/)
```
Comment (alvations) [inline README.md]: Should we allow repeated links in different section?

Comment (thiakx) [reply README.md]: Let's have no repeated links, keep the list clean and simple

File: README.md (hunk 1 above)
Comment (thiakx) [reply README.md:39]: Ok

**Comment by thiakx** [comment]
Added content under new ### Language section as suggested by @alvations. Closing PR

**Pull request closed without merging**